### ISSUE TITLE
Suggestion: Overlay images

### ISSUE THREAD

**Issue by vicmarcag** [oripa/oripa#173]
Hi!

I have a suggestion that I feel it can be very useful when origamists try to decode a folding sequence of a CP developed by other artists. I think it could be very interesting to have an option to overlay an image (i.e., the final CP), so we can find references and put folding sequences over it.

What do you think?

**Comment by Ooouch1** [comment]
Hi.

If it is hard to find references mathematically, we can measure physically (or count the pixels) and use Reference Finder (https://finder.origami.tools/).

Personally, overlaying is not so attractive to me because I have had such difficulties only few times. I think approximating reference on ORIPA is as difficult as finding it mathematically.

Thank you for the suggestion. Let us know if you get new idea!

**Comment by vicmarcag** [comment]
Yes, I use ReferenceFinder as well to find the initial references. However, my suggestions was not entirely focused on that, but in the development of the process to get the rest of the creases given the reference. Currently, I do that on a printed copy, but I think it could be very useful to overlay the final CP image to ease this process and to find a folding sequence

**Comment by Ooouch1** [comment]
I understand but still I don't agree or don't like the way... It sounds like you try to find critical points and creases by coincidences.  It may work if the crease pattern is on 22.5 degree system (or 15 degree system) and you have got the initial reference, but I think mathematical precise way is not so difficult in many cases for such crease patterns. It is an exercise of geometry which we learn in junior high school and high school, at least in Japan.

  We are working on improving line input functionality, and the priority of implementing your suggestion would not be high. 
Sorry.

Note:
  Overlaying an image is not difficult to program (draw the image first and then draw the user inputs) but maybe it requires users to satisfy some restrictions, for example, user may have to trim the outside of the crease pattern in the image. Another restriction is the dimension of the image; the image should be a square to fit the default contour of paper.

**Comment by vicmarcag** [comment]
It's OK! Maybe I will try to implement it myself in a separate fork :)
